components/mosquitto/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [2.0.20~1](https://github.com/espressif/esp-protocols/commits/mosq-v2.0.20_1)
```

Comment:
You also removed multiple files. Is it something users should be informed about?

Reply:
Yes, you're right. I removed those files after the bump commit so it didn't appear in the changelog.

Updated, Thanks!